composer.json

```diff
@@ -19,17 +19,17 @@
         "php": ">=7.0.8",
         "composer/installers": "^1.2",
         "cweagans/composer-patches": "^1.6.5",
-        "drupal-composer/drupal-scaffold": "^2.5",
+        "drupal/core-composer-scaffold": "8.8.x-dev",
+        "drupal/core-recommended": "^8.7.0",
         "drupal/console": "^1.0.2",
-        "drupal/core": "^8.7.0",
         "drush/drush": "^9.0.0",
         "vlucas/phpdotenv": "^2.4",
         "webflo/drupal-finder": "^1.0.0",
         "webmozart/path-util": "^2.3",
         "zaporylie/composer-drupal-optimizations": "^1.0"
     },
     "require-dev": {
-        "webflo/drupal-core-require-dev": "^8.7.0"
+        "drupal/dev-dependencies": "^8.7.0"
```

Comment:
drupal/dev-dependencies is marked as abandoned on packagist. should use drupal/core-dev instead?

Reply:
Yeah it was renamed